__tests__/serverjs/cards.test.js

```diff
@@ -126,21 +128,21 @@ test('cardFromId returns only selected fields', () => {
 test('cardFromId returns a placeholder card object when given a nonexistent ID', () => {
   expect.assertions(1);
   const _id = 'not real';
-  var expected = placeholderCard;
+  const expected = placeholderCard;
   expected._id = _id;
-  var promise = carddb.initializeCardDb(fixturesPath, true);
-  return promise.then(function() {
+  const promise = carddb.initializeCardDb(fixturesPath, true);
+  return promise.then(() => {
     const result = carddb.cardFromId(_id);
     expect(result).toEqual(expected);
   });
 });
 
 test('getCardDetails returns a well-formed card object', () => {
   expect.assertions(1);
-  const _id = _RankleMasterofFixtures._id;
-  let expected = _RankleMasterofFixtures;
-  var promise = carddb.initializeCardDb(fixturesPath, true);
-  return promise.then(function() {
+  const { _id } = _RankleMasterofFixtures;
+  const expected = _RankleMasterofFixtures;
+  const promise = carddb.initializeCardDb(fixturesPath, true);
```

Comment:
For all these couldn't we just do `await carddb.initializeCardDb(fixturesPath, true)` to avoid callbacks? Also is that something we can move into a setup block instead of duplicating it in each test?

Reply:
yea, that's probably a good update...

Reply:
but maybe on a different PR just to keep things simple...